test/integration/dynamic-routing/pages/index.js

```diff
@@ -5,10 +5,14 @@ const Page = () => {
   return (
     <div>
       <h3>My blog</h3>
-      <Link href="/[name]" as="/post-1">
+      <Link as="/post-1" href="/[name]">
```

Comment:
Why was this reversed?

Reply:
Got flipped around while updating tests, reverted